docs/howto/tracing.md

```diff
@@ -1,16 +1,57 @@
-(trace-the-charm-code)=
-# Trace the charm code
+(how-to-tracing)=
+# How to trace charm code
+## Introduction
 
-`ops[tracing]` provides the first party charm tracing library,
-`ops.tracing.Tracing`, allowing you to observe and instrument your charm's
-execution using OpenTelemetry.
+This document describes how to trace your charm code and send the trace data to the [Canonical Observability Stack](https://documentation.ubuntu.com/observability/).
 
-Refer to the {ref}`ops_tracing` reference for the canonical usage example, configuration
-options, and API details.
+### Why trace your charms
+
+Observability transforms a Juju deployment from a black box to a real-time system.
```

Comment:
I'm not sure the how-to needs to answer "why trace your charm" in general. But if we do include such a section, I don't love this sentence. Juju's real-time-ness is unrelated to whether it's observed or not. I don't think it's entirely fair to say that it's a black box without observability, either.

I think if we do include this section we should have a few concrete examples of problems that we're trying to solve (and ideally also some hints at what we are not trying to solve, like profiling).

However, it feels like the explanation is already working at answering this question, so we don't need it here too.